crates/gpui/src/text_system/line_wrapper.rs

```diff
@@ -49,9 +49,17 @@ impl LineWrapper {
                     continue;
                 }
 
-                if prev_c == ' ' && c != ' ' && first_non_whitespace_ix.is_some() {
-                    last_candidate_ix = ix;
-                    last_candidate_width = width;
+                if Self::is_word_char(c) {
```

Comment:
Actually, an idea:

Why not just flip this conditional, and check for CJK characters? 
https://stackoverflow.com/questions/1366068/whats-the-complete-range-for-chinese-characters-in-unicode

This seems more robust to other kinds of changes we could have, rather than positioning CJK as the default.

And honestly, we could probably just do a single check for the `4E00-9FFF` range of unicode for now.  

Curious what you think!

Reply:
Because in wrapping we pay more attention to **keeping words**, while other non-word words should be wrapped directly by char.

Reply:
And the scenes in CJK are actually more complex, containing not only ordinary text, but also a lot of special symbols. It is divided into many Unicode Ranges.

Reply:
Ahhh, ok. Thanks for the explanation.